### ISSUE TITLE
[BugReport] schedule did not update

### ISSUE THREAD

**Issue by whklhh** [misenhower/splatoon3.ink#84]
The home page, which should shows the schedules of regular, bankara and X-match, is empty.
It seems like schedule.json has stopped update 2*5 hours ago.

**Comment by Kade-N** [comment]
This might be why https://bsky.app/profile/joythedatanerd.bsky.social/post/3lcwgjshxuk2n

> Nintendo just nuked programmatic access to SplatNet. I’ll keep my eye out for when people smarter than me solve the issue, but if it’s not solvable it’ll probably mean I’ll move on from developing things for the community, too much effort nuked by Nintendo.
> Nintendo has held a sword of Damocles above us for a while with NSO, threatening to shut it all down. Tonight they finally forced us all to upgrade, which kills every tool that accesses NSO like [splat.top](https://splat.top/), nxapi, any [stat.ink](https://stat.ink/) uploading client, and more. It’s pretty awful

**Comment by misenhower** [comment]
Yep, that is exactly why. There were some changes in the NSO app and this site isn't currently able to authenticate to Nintendo's servers to retrieve data.

nxapi project members are actively working toward fixing this though, and it sounds like they have a couple different workarounds in mind that might help. Hopefully we can get this fixed and get the site back online soon!

**Comment by misenhower** [comment]
I just pulled a token manually from my phone, so that will fix the updater for about 2 hours. No guarantees since it's a manual process but I'll try to update this a couple times a day or so for now.

**Comment by whklhh** [comment]
I see. Thank you very much~

**Comment by Bacongitgit** [comment]
> I just pulled a token manually from my phone, so that will fix the updater for about 2 hours. No guarantees since it's a manual process but I'll try to update this a couple times a day or so for now.

Hello, I would like to ask how to export tokens from non rooted Android phones or non jailbroken iPhone？

**Comment by misenhower** [comment]
@Bacongitgit You can use a tool like mitmproxy or Fiddler to intercept traffic from your phone. You'll need to install a root certificate to be able to decrypt HTTPS traffic.

**Comment by Bacongitgit** [comment]
@misenhower Thanks :)

**Comment by NBSgamesAT** [comment]
I will have to bump this. Data didn't seem to update again. I just wanted to say something to let you know that authorization with Nintendo's server seems to fail again
